ZenPacks/zenoss/ZenPackLib/lib/base/ZenPack.py

```diff
@@ -285,16 +285,33 @@ def template_changed(self, app, existing, new_spec):
         return diff
 
     def create_device_class(self, app, dcspec):
-        ''''''
+        """Return existing or created device class"""
+
+        def handle_plugins(d_class, value, overwrite):
```

Comment:
handle_plugins shouldn't be an inner function. It's only called once within the function.